docs/src/components/Container/ComponentsContainer.js

```diff
@@ -40,9 +32,16 @@ const ComponentsContainer = ({ children, pageTitle, relativePagePath, tabs, page
     const pages = pagePath.slice(0, pagePath.length - 1);
     const path = `${pages.join('/')}/${nextTabSlug}/`;
     navigate(path, { state: { animation: false } });
-    setActiveIndex(tabIndex);
   };
 
+  const activeTabIndex = React.useMemo(() => {
+    const foundIndex = tabsList?.length
```

Comment:
Please update this variable name `foundIndex` to be more appropriate

Reply:
ok